Virtual Keyboard/Assets/Scripts/SlidingList.cs

```diff
@@ -25,21 +16,15 @@ class InnerListItem {
 	GameObject listObject;
 	Vector3 baseScale;
 
-	int selectedIndex;
+	int selectedIndex = 0;
 	bool initialized = false;
 	bool selector_at_top = true;
 
 	// Use this for initialization
 	void Start () {
-		myInitialize ();
-	}
-
-	void myInitialize(){
 		if (!initialized) {
 			baseScale = transform.Find ("ListOfItems").parent.transform.localScale;
```

Comment:
gameObject.transform.localScale would be equivalent would it not? This would circumvent the search though children objects.